README.md

```diff
@@ -32,6 +32,31 @@ Then you have to call the constructor with the correct dev/prod variables
 
 `var metrics = new Auth0Metrics('segmentKey', 'dwhEndpoint', 'website');`
 
+### Loader
+
+A script that will asynchronously load `auth0-metrics.js` is also provided.
+
+```html
+<script src="auth0-metrics-loader.js"></script>
+<script>
+  metricsLib.load({
+    segmentKey: 'segmentKey',
+    dwhEndpoint: 'dwhEndpoint',
+    label: 'website'
+  });
+</script>
+```
```

Comment:
write the 3 of them then :) the difference between the 2 is the stub. While the 3rd will throw if one of the parameters (segmentKey, dwhEndpoint) is missing, the stub wont and it's safer IMHO.